examples/trusted_information_retrieval/backend/build.rs

```diff
@@ -14,30 +14,14 @@
 // limitations under the License.
 //
 
-use std::path::Path;
-
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let proto_path = Path::new("../../../examples/trusted_information_retrieval/proto");
-    let trusted_information_retrieval_path = proto_path.join("trusted_information_retrieval.proto");
-    let database_path = proto_path.join("database.proto");
-
-    // Tell cargo to rerun this build script if the proto file has changed.
-    // https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath
-    println!(
-        "cargo:rerun-if-changed={}",
-        trusted_information_retrieval_path.display()
-    );
-    println!("cargo:rerun-if-changed={}", database_path.display());
-
-    tonic_build::configure()
-        .build_client(false)
-        .build_server(true)
-        .compile(
-            &[
-                trusted_information_retrieval_path.as_path(),
-                database_path.as_path(),
-            ],
-            &[proto_path],
-        )?;
+    oak_utils::compile_server_proto(
+        "../../../examples/trusted_information_retrieval/proto",
+        "trusted_information_retrieval.proto",
```

Comment:
Shouldn't the second argument be a `Vec` rather? Or is it better to have separate invocations like this?

Reply:
I had some issues before where calling `prost-build` `compile` twice would overwrite the files generated by the first call. Does tonic handle this differently?